code/modules/mob/living/carbon/superior_animal/nanobot/nanobot.dm

```diff
@@ -73,6 +73,12 @@
 	Console = new /obj/item/modular_computer/console/preset/nanobot(src)
 	update_icon()
 
+/mob/living/carbon/superior_animal/nanobot/rejuvenate()
+	..()
+	//We trgain are consol and radio if revived!
+	Radio = new/obj/item/device/radio(src)
+	Console = new /obj/item/modular_computer/console/preset/nanobot(src)
```

Comment:
```suggestion
	Console = new /obj/item/modular_computer/console/preset/nanobot(src)
```
Doesn't that mean that people will be able to 'farm' nanobots by taking out the console, then killing them before reviving the nanobot?